src/Npgsql/Schema/NpgsqlDbColumn.cs

```diff
@@ -176,7 +176,7 @@ public new string ColumnName
         /// <inheritdoc cref="DbColumn.DataTypeName" />
         public new string DataTypeName
         {
-            get => base.DataTypeName;
+            get => base.DataTypeName!;
```

Comment:
And how this could be nullable in .NET?

Reply:
Mostly because there's this sentence in the [docs](https://docs.microsoft.com/en-us/dotnet/api/system.data.common.dbcolumn.datatypename?view=netcore-3.1#System_Data_Common_DbColumn_DataTypeName):

> The name of the data type; otherwise, a null reference (Nothing in Visual Basic) if no value is set.

Maybe there could be a database where that information isn't available or whatever, who knows...